docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -32,17 +32,17 @@ To enable periodical dumps, define the following properties:
     jmx.max-entries=86400
 
 ``dump-tables`` is a comma separated list of Managed Beans (MBean). It specifies
-which MBeans is sampled and stored in memory every ``dump-period``.
-History has limited size of ``max-entries`` of entries. Both ``dump-period``
-and ``max-entries`` have default values of ``10s`` and ``86400`` accordingly.
+which MBeans are sampled and stored in memory every ``dump-period``. History
+has a limited size of ``max-entries`` number of entries. Both ``dump-period``
```

Comment:
Maybe change to break the two properties up in context... something like

```
You can configure the maximum number of history entries with ``max-entries``. It defaults to ``86400``. The time between dumps defaults to ``10s`` and is configured with ``dump-period`` property.
```


The maximum number of entries can be confi history is limited to number of entries, de, ``86400`` by default. It can be configured with ``max-et